pkg/oc/bootstrap/docker/run_self_hosted.go

```diff
@@ -133,6 +134,7 @@ func (c *ClusterUpConfig) StartSelfHosted(out io.Writer) error {
 		return err
 	}
 
+	clientConfig.Host = c.ServerIP + ":8443"
```

Comment:
@deads2k not sure if this is needed, but without this I was not able to get the `--public-hostname` to work. In order to bind the cluster up on the VM, I had to specify the VM IP address...

Reply:
@deads2k also I think I saw somebody on IRC reporting that --public-hostname is broken